bubblewrap.c

```diff
@@ -85,6 +85,9 @@ bool opt_unshare_cgroup = FALSE;
 bool opt_unshare_cgroup_try = FALSE;
 bool opt_needs_devpts = FALSE;
 bool opt_new_session = FALSE;
+#ifdef WARN_IMPLICIT_SESSION_OPT
+bool specified_explicitly_opt_new_session = FALSE;
+#endif
```

Comment:
I'd call this `opt_new_session_set`, and it might as well exist unconditionally, even if it's write-only in the case where no warning is emitted: that would save a lot of `#ifdef` elsewhere.